### PR TITLE
Save profiles when running startLocal

### PR DESCRIPTION
**Pull request by nolanmar511**
Right now, profiles are thrown away when running startLocal. With this change, profiles will be written out to files.

### CODE REVIEW

**Comment by nolanmar511** [comment]
PTAL

File: ts/src/index.ts [hunk 1]
```diff
@@ -148,13 +151,27 @@ export async function startLocal(config: Config = {}): Promise<void> {
     if (!config.disableHeap) {
       const heap = await profiler.profile(
           {name: 'HEAP-Profile' + new Date(), profileType: 'HEAP'});
+      const decodedBytes = Buffer.from(heap.profileBytes as 'string', 'base64');
+      fs.writeFile(
```
Comment (aalexand) [inline ts/src/index.ts]: Would be good to document somewhere in the code where the profiles will get written (current directory? temp directory?)

Comment (nolanmar511) [reply ts/src/index.ts]: Done

**Comment by aalexand** [comment]
Should this PR be closed or submitted?

**Comment by nolanmar511** [comment]
I think I'll close this. If startLocal does save profiles, it should probably be only when the user specifies this.

**Pull request closed without merging**